src/python/WMCore/Services/DBS/DBSUtils.py

```diff
@@ -137,3 +139,28 @@ def urlParams(url):
         if len(vals) == 1:
             rdict[key] = vals[0]
     return rdict
+
+def DBSErrors(dbsUrl):
```

Comment:
I thought I had seen an unit test for this new method (when I suggested to mock it - after running a real unit test and ensuring it works as expected). Did it get lost during the squash process?

Reply:
there were no test provided initially, I just add new integration unit test for it.. Regarding mocking itself, I'm not sure where and how to do it. If it is required please elaborate on this request more. For instance, I can write new function in src/python/WMQuality/Emulators/DBSUtils/DBSUtils.py to simulate output of DBS `/errors` end-point (it is a list of dicts) but then I don't know where to use it.